warrior/Framework/ClassUtils/WNetwork/warrior_cli_class.py

```diff
@@ -277,6 +277,8 @@ def update_resp_ref_to_repo(self, details_dict, resp_key_list, i,
                 pNote("Portion of response saved to the data "
                       "repository with key: '{0}.{1}.{2}' and value: '{3}'"
                       .format(session_id, title_row, resp, temp_resp[resp]))
+        except Exception as e:
+            print_error("Found exception: {}".format(e))
```

Comment:
Hi @navvrnd 
This error message is too vague that it doesn't tell the user what went wrong in the code
please provide information for user to correct from the exception

Reply:
Hi @kaheichan made the changes. Please check and approve.